### ISSUE TITLE
Links towards issues connecting PRs are downcased

### ISSUE THREAD

**Issue by EmCousin** [ZenHubIO/support#1240]
Hi,

When I open a PR and want to connect it with an issue, the link to the issue is downcased.
![image](https://user-images.githubusercontent.com/9846489/30205159-5121eb82-9488-11e7-8eca-892b24a6d325.png)

Though the link still redirects me to the issue, this is particularly inconvenient as :
- When copy-pasting the given link in some issue/PR description or comment, GitHub cannot shorten it properly
- The reference to the PR cannot be seen in the issue.

I assume Github made the URL shortening system case-sensitive for a reason, but I am not sure why. Is it possible for ZenHub to give the exact case-sensitive URL?

**Comment by kendramoroz** [comment]
@EmCousin good catch! Issuing this up as a 🐛  for the team and bumping into Known Issues.
